pkg/daemon/update_test.go

```diff
@@ -774,51 +774,20 @@ func TestOriginalFileBackupRestore(t *testing.T) {
 	testDir, cleanup := setupTempDirWithEtc(t)
 	defer cleanup()
 
-	// Write a normal file as a control to make sure normal case works
+	// Write a file in the /tmp dir to test whether orig files are selectively preserved
 	controlFile := filepath.Join(testDir, "control-file")
 	err := os.WriteFile(controlFile, []byte("control file contents"), 0755)
 	assert.Nil(t, err)
 
-	// Back up that normal file
+	// Back up the tmp file
 	err = createOrigFile(controlFile, controlFile)
 	assert.Nil(t, err)
 
-	// Now try again and make sure it knows it's already backed up
+	// Now try again and make sure it knows it's already backed up if it should be
 	err = createOrigFile(controlFile, controlFile)
 	assert.Nil(t, err)
 
-	// Restore the normal file
```

Comment:
question: Are these tests no longer needed or do they now fail with this new change?

Reply:
I think I am deleting those because they now fail with this new change.  This is because the tests create files in the /tmp folder, try to create orig files for those and see if that is successful by restoring form the orig files. But the current workaround stops the orig file preservation for the files in the /tmp folder as they are not the original files shipped with the disk, so the tests are no longer valid. However, I have also tested the  current createOrig' s functionality in preserving the files by putting a forceCreate variable in, so that when running the test, the origs are forcedly created for these /tmp files. After verifying its functionality, I removed those variables as well as the tests.

Reply:
To give a bit more context, I think these tests aren't really needed anymore since we are now (properly) not preserving anything written to /tmp, since they shouldn't have been preserved in the first place.

We could instead test preservation of `/etc` specifically but it is a bit harder to do so via a unit test and may modify the user's system, so we decided to remove this test for now. If there is a simple way to test that that we're missing, would also be happy to take suggestions!